mne/tests/test_annotations.py

```diff
@@ -834,6 +834,11 @@ def test_annotations_slices():
                         description=EXPECTED_DESCS.copy(),
                         orig_time=None)
 
+    # Indexing returns a copy. So this has no effect in annot
+    annot[0]['onset'] = 42
```

Comment:
Does it surprise you that we cannot modify it? I know that this is exactly why we return a copy. But when written like this I find it surprising.

Reply:
I can live with that. It's fine to write:

annot.onset[0] = 42

Reply:
then this should be in the tuto